build.gradle.kts

```diff
@@ -80,27 +89,17 @@ dependencies {
 }
 
 application {
-    // Changing this as the deprecation suggests will break the ShadowJar plugin
-    // Alternative code for when it gets updated:
-    // mainClass.set("com.github.devcordde.devcordbot.LauncherKt")
-    mainClassName = "com.github.devcordde.devcordbot.LauncherKt"
+    mainClass.set("com.github.devcordde.devcordbot.LauncherKt")
 }
 
 tasks {
-    shadowJar {
-        transform(com.github.jengelman.gradle.plugins.shadow.transformers.ServiceFileTransformer::class.java)
-    }
-
-    compileKotlin {
-        kotlinOptions.jvmTarget = "14"
-    }
-
-    compileTestKotlin {
-        kotlinOptions.jvmTarget = "14"
-    }
-
-    jar {
-        archiveClassifier.value = "original"
+    withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
+        kotlinOptions {
+            jvmTarget = "14"
```

Comment:
it is since 4 a few hrs ago as Kotlin 1.5 released and my java 15 jdk had issues. But the 1.5 gradle plugin is not in the Gradle repos yet so we need to wait for that

Reply:
I upgraded it to Kotlin 1.5 and targetted java 16